src/GDBone2D.cpp

```diff
@@ -102,8 +95,8 @@ Vector2 GDBone2D::get_bone_position() {
 void GDBone2D::set_bone_position(Vector2 new_pos) {
 	if (boneData == nullptr) return;
 
-	boneData->animationPose.x = new_pos.x;
-	boneData->animationPose.y = new_pos.y;
+	boneData->animationPose.x = new_pos.x / get_global_transform().get_scale().x;
```

Comment:
Dividing by the global scale allows us to specify offset in godot coordinates and not the scaled Dragonbones coordinates. Doing this allows us to create the `set_bone_global_position` in Godot coordinates as well